include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -912,4 +912,21 @@ def Torch_OverwriteTensorOp : Torch_Op<"overwrite.tensor", [
   }];
 }
 
+def Torch_RngUniformOp : Torch_Op<"rng_uniform", [
+    AllowsTypeRefinement,
+    HasValueSemantics,
+  ]> {
+  let summary = "`uniform op : (Tensor, float, float, Generator?) -> (Tensor)`";
```

Comment:
Can you explain what you mean by having this op "as the building
block of other torch rng ops"?

Reply:
I meant other ops like `bernoulli` could be decomposed to this op.